spec/jobs/email_notes_to_admins_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe EmailNotesToAdminsJob, type: :job do
+  include ReservationHelper
+  shared_examples 'admin email' do |*scopes|
+    it 'sends emails' do
+      res = mock_reservation
+      stub_scope_chain([res], *scopes)
+      allow_any_instance_of(Array).to receive(:update_all)
+        .with(notes_unsent: false).and_return([])
+      expect(AdminMailer).to \
+        receive_message_chain(:notes_reservation_notification, :deliver_now)
+      described_class.perform_now
+    end
+    it 'gets the appropriate reservations' do
+      expect(Reservation).to receive_message_chain(*scopes).and_return([])
+      described_class.perform_now
+    end
+    it 'unsets the notes_unsent flag' do
+      res = mock_reservation
+      stub_scope_chain([res], *scopes)
+      allow(AdminMailer).to \
+        receive_message_chain(:notes_reservation_notification, :deliver_now)
+      expect_any_instance_of(Array).to receive(:update_all)
```

Comment:
Why can't we we save `res_array = [res]` and then just `expect(res_array).to receive(:update_all)...`?